packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -308,7 +286,7 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
                                                                     'Length of token name must be between 2 and 32 characters.'
                                                                 )}
                                                                 required
-                                                                error={touched.token_name && errors.token_name}
+                                                                error={errors.token_name}
```

Comment:
```suggestion
 error={touched.token_name && errors.token_name ? errors.token_name : undefined}
```

We can try writing like this if we're getting a type error related to boolean